DevOps.Util/HelixServer.cs

```diff
@@ -99,15 +112,43 @@ public async ValueTask GetHelixPayloads(string jobId, List<string> workItems, st
                     if (string.IsNullOrEmpty(workItemInfo.PayloadUri))
                         continue;
 
-                    await DownloadWorkitemFiles(workItemInfo.WorkItemId!, workItemInfo.PayloadUri).ConfigureAwait(false);
+                    string itemDir = await DownloadWorkitemFiles(workItemInfo.WorkItemId!, workItemInfo.PayloadUri).ConfigureAwait(false);
+
+                    Console.WriteLine();
+                    Console.WriteLine($"----- To execute: {workItemInfo.WorkItemId} -----");
+                    Console.WriteLine();
+                    string setPrefix = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "set " : "";
+                    Console.WriteLine($"{setPrefix}HELIX_CORRELATION_PAYLOAD={correlationDir}");
+                    Console.WriteLine($"{setPrefix}HELIX_PYTHONPATH=echo skipping python");
```

Comment:
you'll need to wrap this in quotes on !Windows since otherwise we'll get this:

```bash
alexander:~/dev$ HELIX_PYTHONPATH=echo skipping python
zsh: command not found: skipping
```